en/asset/cache-manager.md

```diff
@@ -34,9 +60,9 @@ resources.load('images/background/texture', Texture2D, function (err, texture) {
 });
 ```
 
-## Query temporary files
+### Querying temporary files
 
-After a resource is downloaded locally, it may be stored as a temporary file in a temporary directory. The Cache Manager provides a `tempFiles` interface to query all resources downloaded to the temporary directory, which you can do by passing in the original path of the resource, for example:
+When a resource is downloaded locally, it may be stored as a temporary file in a temporary directory. The cache manager provides the ``tempFiles`` interface to query all resources downloaded into the temporary directory, and developers can do so by passing in the original path of the resource.
```

Comment:
```suggestion
When a resource is downloaded locally, it may be stored as a temporary file in a temporary directory. The cache manager provides the `tempFiles` interface to query all resources downloaded into the temporary directory, and developers can do so by passing in the original path of the resource.
```